BaseClasses.py

```diff
@@ -1389,6 +1390,14 @@ def write_option(option_key: str, option_obj: Options.AssembleOptions) -> None:
 
             AutoWorld.call_all(self.multiworld, "write_spoiler", outfile)
 
+            precollected_items = [f'{item.name} ({self.multiworld.get_player_name(item.player)})'
+                                  if self.multiworld.players > 1
+                                  else item.name
+                                  for item in chain.from_iterable(self.multiworld.precollected_items.values())]
+            if precollected_items:
+                outfile.write('\n\nPrecollected Items:\n\n')
```

Comment:
these should all be `"` not `'`. i know you did it to be stylistically consistent with the code in this area but it's already inconsistent. new code should use the new style and you can either "fix" some nearby code as part of this or ignore it.